### ISSUE TITLE
"unsupported layout" error

### ISSUE THREAD

**Issue by arturjaworski** [tadija/AEXML#95]
When running `swift build` I'm getting following error:

```
$ swift build
error: the package has an unsupported layout, unexpected source file(s) found: Packages/AEXML-4.0.1/Tests/AEXMLTests.swift
fix: move the file(s) inside a module
```

**Comment by StrangeDays** [comment]
mkdir Packages/AEXML-4.0.1/Tests/AEXMLTests  
mv Packages/AEXML-4.0.1/Tests/AEXMLTests.swift Packages/AEXML-4.0.1/Tests/AEXMLTests
swift build

**Comment by tadija** [comment]
Thanks for reporting, this should be fixed now in 4.0.2.
